FLIP2PDF/Acknowledgements.html

```diff
@@ -0,0 +1,185 @@
+<p>8. Litigation Any litigation relating to this License may be brought only in the courts of a jurisdiction where the defendant maintains its principal place of business and such litigation shall be governed by laws of that jurisdiction, without reference to its conflict-of-law provisions. Nothing in this Section shall prevent a party’s ability to bring cross-claims or counter-claims.</p>
+<p>9. Miscellaneous This License represents the complete agreement concerning the subject matter hereof. If any provision of this License is held to be unenforceable, such provision shall be reformed only to the extent necessary to make it enforceable. Any law or regulation which provides that the language of a contract shall be construed against the drafter shall not be used to construe this License against a Contributor.</p>
+<p>10. Versions of the License</p>
+<p>10.1. New Versions Mozilla Foundation is the license steward. Except as provided in Section 10.3, no one other than the license steward has the right to modify or publish new versions of this License. Each version will be given a distinguishing version number.</p>
+<p>10.2. Effect of New Versions You may distribute the Covered Software under the terms of the version of the License under which You originally received the Covered Software, or under the terms of any subsequent version published by the license steward.</p>
+<p>10.3. Modified Versions If you create software not governed by this License, and you want to create a new license for such software, you may create and use a modified version of this License if you rename the license and remove any references to the name of the license steward (except to note that such modified license differs from this License).</p>
+<p>10.4. Distributing Source Code Form that is Incompatible with Secondary Licenses If You choose to distribute Source Code Form that is Incompatible with Secondary Licenses under the terms of this version of the License, the notice described in Exhibit B of this License must be attached.</p>
+<p>Exhibit A - Source Code Form License Notice This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, you can obtain one at <a href="http://mozilla.org/MPL/2.0">http://mozilla.org/MPL/2.0</a>.</p>
+<p>If it is not possible or desirable to put the notice in a particular file, then You may include the notice in a location (such as a LICENSE file in a relevant directory) where a recipient would be likely to look for such a notice.</p>
+<p>You may add additional accurate notices of copyright ownership.</p>
+<p>Exhibit B - “Incompatible with Secondary Licenses” Notice This Source Code Form is “Incompatible with Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.</p>
+<h2>WK&lt;HTML&gt;toPDF</h2>
```

Comment:
can you confirm this is the correct rendering on this header?
<img width="1175" alt="Screen Shot 2023-07-27 at 7 45 19 AM" src="https://github.com/datalogics/datalogics.github.io/assets/1760932/734ef409-927b-44dd-a7a0-00c61dd47d22">

Reply:
https://wkhtmltopdf.org/

uses the same rendering, so I believe this was intentional.